src/main/react/server/middlewares/frontendMiddleware.js

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable global-require */
+
+/**
+ * Front-end middleware
+ */
+module.exports = (app, options) => {
+  const isProd = process.env.NODE_ENV === 'production';
```

Comment:
I'm curious how env var this will get set when running in production mode - i.e. when the plugin is release,

Reply:
Actually that's for testing the built files rather than a release version. When it's started as production, the project is serving the built files with [express.js](https://github.com/expressjs/express), thus we could test the built files(I guess). And this may not help too much but still a bit necessary. 
I suggest a clean period after we finish all the features and workflows, and in that stage we could minimize the project. Or shall we minimize it right now?

Reply:
This can be found in the **addProdMiddlewares.js**, it's using express to serving the built files.
`module.exports = function addProdMiddlewares(app, options) {
  const publicPath = options.publicPath || '/';
  const outputPath = options.outputPath || path.resolve(process.cwd(), 'build');

  // compression middleware compresses your server responses which makes them
  // smaller (applies also to assets). You can read more about that technique
  // and other good practices on official Express.js docs http://mxs.is/googmy
  app.use(compression());
  app.use(publicPath, express.static(outputPath));

  app.get('*', (req, res) =>
    res.sendFile(path.resolve(outputPath, 'index.html')),
  );
};
`

Reply:
I see. I think this is fine